### ISSUE TITLE
Line feeds at the end of Dockerfiles are treated as errors

### ISSUE THREAD

**Issue by UnlikeMars** [hadolint/hadolint#463]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### The problem
When working on Dockerfiles with Intellij IDEA, line feeds are automatically inserted at the end of files. Removing tailing line feeds will cause the IDE to reinsert them. This is the IDEs default behavior which makes it difficult to integrate Hadolint. Hadolint treats line feeds at the end of Dockerfiles as errors. A single line feed at the end of a file IMHO shouldn't be treated as an error.

### Expected behavior
Single line feeds which are automatically inserted by IDEs at the end of Dockerfiles are ignored.

### Actual behavior
Single line feeds which are automatically inserted by IDEs at the end of Dockerfiles are treated as errors:

`/dev/stdin:3:40 unexpected carriage return expecting '\', a new line followed by the next instruction, or at least one space`

### Steps to reproduce the behavior
Run `docker run --rm -i hadolint/hadolint < Dockerfile` in cmd.exe

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.18.0-6-ga0d655d
```

**Dockerfile** (if relevant)
_Please note the line break after the fourth line of code._
```
Dockerfile
FROM openjdk:11-jdk-slim
COPY build/libs/*.jar app.jar
ENTRYPOINT ["java", "-jar", "/app.jar"]

```

**Additional environment details (OS, stack version, etc.)**
- Windows 10
- Intellij IDEA 2020.2.1

**Comment by lorenzo** [comment]
Yeah, a known bug in windows. Sorry I have not had the time to fix this. You can use a previous version in the meantime

**Comment by testworksau** [comment]
I was a bit confused by the "A previous version" comment as I tried the current previous version (1.17.7) which didn't work.

For the benefit of other people:
1.17.6 (v1.17.6-0-gc918759) is the last version without this bug
1.17.7 (v1.17.7-0-gd807e87) is the first version with this bug
